Bio/SeqIO/SeqXmlIO.py

```diff
@@ -262,9 +283,13 @@ def startDBRefElement(self, attrs):
                 elif localname == "id":
                     ID = value
                 else:
-                    raise ValueError("Unexpected attribute '%s' found for DBRef element", key)
+                    raise ValueError(
+                        "Unexpected attribute '%s' found for DBRef element", key
```

Comment:
Shouldn't this be `% key` instead of `, key`?

Reply:
Yes, good catch